server/server/Routes/signup.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const router = express.Router();
+const { body } = require("express-validator");
+const userController = require("../controllers/userController");
+
+// Define the sign-up route
+router.post(
+  "/signup",
+
+  // Validation rules
+  body("username").isLength({ min: 5 }),
```

Comment:
i think we should include a custom message with each validation rule to be sent in response, like:

```suggestion
  body("username", "Username should be at least 5 characters").isLength({ min: 5 }),
```